py_trees/console.py

```diff
@@ -35,6 +35,25 @@
 import os
 import sys
 
+
+##############################################################################
+# Special Characters
+##############################################################################
+
+def correct_encode(original: str, replacement: str, encoding: str = sys.stdout.encoding):
+    try:
+        original.encode(encoding)
+    except UnicodeError:
+        return replacement
+    return original
+
+
+LIGHTNING_BOLT = correct_encode(u'\u26A1', "SYNC")
```

Comment:
Let's use lower case here. Breaks certain style guides I know, but stays consistent to what this module started with (all the colours are lower case) many years ago.

Reply:
Done